### PR TITLE
Fixed some documentation links and typos

### PR DESCRIPTION
**Pull request by thalbern**
## Overview
- Fixed broken hand tracking link in readme
- replaced all links to .html with relative .md links
- replaced api doc link with xref
- fixed typo in code doc
- replaced the word nuke with clear in code comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ MRTK-Unity is a Microsoft-driven project that provides a set of components and f
...
+| [![Getting Started and Documentation](Documentation/Images/MRTK_Icon_GettingStarted.png)](Documentation/GettingStartedWithTheMRTK.md)<br/>[Getting Started](Documentation/GettingStartedWithTheMRTK.md)| [![Getting Started](Documentation/Images/MRTK_Icon_ArchitectureOverview.png)](Documentation/Architecture/Overview.md)<br/>[MRTK Overview](Documentation/Architecture/Overview.md)| [![Feature Guides](Documentation/Images/MRTK_Icon_FeatureGuides.png)](Documentation/README_Button.md)<br/>[Feature Guides](Documentation/README_Button.md)| [![API Reference](Documentation/Images/MRTK_Icon_APIReference.png)](xref:Microsoft.MixedReality.Toolkit)<br/>[API Reference](xref:Microsoft.MixedReality.Toolkit.Boundary)|
```
Comment (thalbern) [inline README.md]:
```suggestion
| [![Getting Started and Documentation](Documentation/Images/MRTK_Icon_GettingStarted.png)](Documentation/GettingStartedWithTheMRTK.md)<br/>[Getting Started](Documentation/GettingStartedWithTheMRTK.md)| [![Getting Started](Documentation/Images/MRTK_Icon_ArchitectureOverview.png)](Documentation/Architecture/Overview.md)<br/>[MRTK Overview](Documentation/Architecture/Overview.md)| [![Feature Guides](Documentation/Images/MRTK_Icon_FeatureGuides.png)](Documentation/README_Button.md)<br/>[Feature Guides](Documentation/README_Button.md)| [![API Reference](Documentation/Images/MRTK_Icon_APIReference.png)](xref:Microsoft.MixedReality.Toolkit)<br/>[API Reference](xref:Microsoft.MixedReality.Toolkit)|
```

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ MRTK-Unity is a Microsoft-driven project that provides a set of components and f
 
 # Getting started with MRTK
 
-| [![Getting Started and Documentation](Documentation/Images/MRTK_Icon_GettingStarted.png)](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/GettingStartedWithTheMRTK.html)<br/>[Getting Started](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/GettingStartedWithTheMRTK.html)| [![Getting Started](Documentation/Images/MRTK_Icon_ArchitectureOverview.png)](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Architecture/Overview.html)<br/>[MRTK Overview](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Architecture/Overview.html)| [![Feature Guides](Documentation/Images/MRTK_Icon_FeatureGuides.png)](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Button.html)<br/>[Feature Guides](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Button.html)| [![API Reference](Documentation/Images/MRTK_Icon_APIReference.png)](https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.html)<br/>[API Reference](https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.html)|
```
Comment (keveleigh) [inline README.md]: Do you think we might want to keep these links in the base README as explicit `github.io` links, so you end up on the actual docs site even if you start from the README on GitHub?

Comment (thalbern) [reply README.md]: i'm not sure - if you do that you will point to the mrtk_development doc version - when keeping them relative we point to the version specific ones

Comment (keveleigh) [reply README.md]: Oh, good point!

**Pull request merged**